CHANGELOG

```diff
@@ -32,6 +32,7 @@
 * iATS Payments: Add support for Customer Code payment method [molbrown] #3611
 * HPS: Add Google Pay support [MSmedal] #3597
 * Adyen: Parse appropriate message for 3DS2 authorization calls [britth] #3619
+* Cybersource: Add error detail [schwarzgeist] ####
```

Comment:
Now that you've created a PR for this, you can fill out the number:
```suggestion
* Cybersource: Add error detail [schwarzgeist] #3629
```

Reply:
https://github.com/activemerchant/active_merchant/blob/63233b7e6c85f785d1636a6da0a1b42eda1ea2bd/CHANGELOG#L35